bigbluebutton-html5/imports/api/2.0/users/server/eventHandlers.js

```diff
@@ -16,3 +17,4 @@ RedisPubSub.on('UserEmojiChangedEvtMsg', handleEmojiStatus);
 RedisPubSub.on('SyncGetUsersMeetingRespMsg', handleGetUsers);
 RedisPubSub.on('GuestsWaitingForApprovalEvtMsg', handleGuestsWaitingForApproval);
 RedisPubSub.on('GuestApprovedEvtMsg', handleGuestApproved);
+RedisPubSub.on('UserEjectedFromMeetingEvtMsg', handleKickUser);
```

Comment:
I'd recommend using past tense in the filename and function name. `handleUserKicked` for example. Otherwise it will be confusing when we are sending a message towards akka-apps (i.e. kicked *from* html5)